shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/RealtimePatterns.kt

```diff
@@ -23,6 +23,11 @@ sealed class RealtimePatterns : Comparable<RealtimePatterns> {
     abstract val alertsHere: List<Alert>?
     abstract val hasSchedulesToday: Boolean
 
+    val hasMajorAlerts
+        get() = run {
```

Comment:
Since we can have computed properties in kotlin, I'm still a bit unclear on when and where a function is more or less appropriate. Lately I've been leaning more in the direction of, if it only uses data internal to the class and doesn't need any arguments, make it a computed prop. But if you think a function makes more sense, I wouldn't be opposed to that. I can do a test for sure though.

Reply:
👍 that reasoning makes sense to me! As the person with the least hands-on kotlin experience, happy to adopt that convention.